packages/expo-av/android/CMakeLists.txt

```diff
@@ -43,6 +43,16 @@ target_link_libraries(
         ${LOG_LIB}
         fbjni::fbjni
         ReactAndroid::jsi
-        ReactAndroid::reactnativejni
         android
 )
+if (ReactAndroid_VERSION_MINOR GREATER_EQUAL 76)
```

Comment:
i think `ReactAndroid_VERSION_MINOR` is undefined?

Reply:
Not really, in 0.76 this is what I get when logging `ReactAndroid_VERSION_MINOR is defined: ${ReactAndroid_VERSION_MINOR}"`

<img width="343" alt="image" src="https://github.com/user-attachments/assets/b111cfd2-6fc3-4511-980c-0921e29a08b3">

Reply:
oh i see. it is from cmake: https://cmake.org/cmake/help/latest/variable/PROJECT-NAME_VERSION_MINOR.html#variable:%3CPROJECT-NAME%3E_VERSION_MINOR.
sorry for this false alarm